config/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: prometheus-config
+  namespace: hive
+data:
+  prometheus.yml: |+
+    global:
+      scrape_interval:     15s
```

Comment:
The config I used had `evaluation_interval: 15s` , not sure if that will be something useful.

Reply:
# How frequently to evaluate rules.
  [ evaluation_interval: <duration> | default = 1m ]

That presumably is for alerting rules? I don't think devs will even need this, we should probably just keep it bare minimum for simple prom testing.